modules/modifying-an-existing-ingress-controller.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-an-existing-ingress-controller_{context}"]
+= Modifying an existing Ingress Controller
+
+As a cluster administrator, you can modify an existing Ingress Controller to manage DNS record lifecycle.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* DNS record created.
+
+.Procedure
+
+. Modify an `IngressController` custom resource to set `dnsManagementPolicy`:
```

Comment:
```suggestion
. Modify the existing custom `IngressController` resource to set `dnsManagementPolicy`:
```

Reply:
We don't need to mention word, _existing_ as it is already implied. when we tell the user to modify